Kernel/Arch/Spinlock.h

```diff
@@ -26,6 +26,10 @@ class Spinlock {
 
     [[nodiscard]] ALWAYS_INLINE bool is_locked() const
     {
+        // FIXME: Implement Spinlock on aarch64
```

Comment:
I'd suggest splitting this into a separate commit :)

Reply:
Either way solved.